tasmota/tasmota.ino

```diff
@@ -391,6 +391,43 @@ void setup(void) {
 #endif
 
   TasmotaGlobal.rules_flag.system_init = 1;
+  Serial.println("rulesAfterFlash status:"+Settings.rulesAfterFlash);
+  if (!Settings.rulesAfterFlash){
+      // commands to execute at startup
+      #ifdef defaultSetting_01 
+        ExecuteCommand(defaultSetting_01, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_02 
+        ExecuteCommand(defaultSetting_02, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_03 
+        ExecuteCommand(defaultSetting_03, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_04 
+        ExecuteCommand(defaultSetting_04, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_05 
+        ExecuteCommand(defaultSetting_05, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_06 
+        ExecuteCommand(defaultSetting_06, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_07 
+        ExecuteCommand(defaultSetting_07, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_08 
+        ExecuteCommand(defaultSetting_08, SRC_IGNORE);
+      #endif
+      #ifdef defaultSetting_09 
+        ExecuteCommand(defaultSetting_09, SRC_IGNORE);
+      #endif
+      Settings.rulesAfterFlash=true;
+      SettingsSave(0);
+      AddLog(LOG_LEVEL_INFO, PSTR(D_LOG_RULES " " D_RULE_SETUP_AFTER_FLASHING));
+  }else{
+      Serial.println("\nskipping - Rules setup after flashing firmware\n");
+  }
+
```

Comment:
The above implemntation is a no-go. Performing ExecuteCommand in sequence without any loop access is bound to exception or watchdog. That's where backlog kicks in.

Reply:
without the rulesAfterFlash at each reboot the setup function is processed and this will involve execution of
SettingsSave(0), so it will write the flash at each reboot, or not ?

The execute command for applying rules is a fast process why should it trigger the watchdog ?

What is exactly the new CMND feature field in the template ?

Reply:
CMND in the template is a backlog-style command sequence executed when the template is applied. #11788

Reply:
If my understanding is right, the CMND feature is intended for actions execution or activate/deactivate options but how would I setup a timer based rule triggered by GPIO states ?